google/earthenterprise
# # 1711: Remove FileUnpacker build dir from portable archive

Changes the archive add to only include the packaging directory. This then removes the FileUnpacker build directory from the created archives.

To test:
1. Build portable.
2. Verify that the created archive does not contain the FileUnpacker build directory.
